core/create/CHANGELOG.md

```diff
@@ -60,6 +63,15 @@
     * @dotcom-tool-kit/types bumped from ^2.0.0 to ^2.1.0
     * dotcom-tool-kit bumped from ^2.1.0 to ^2.1.1
 
+### [2.0.1](https://github.com/Financial-Times/dotcom-tool-kit/compare/create-v2.0.0...create-v2.0.1) (2022-05-05)
```

Comment:
**praise**: nice spot - this must have come from here https://github.com/Financial-Times/dotcom-tool-kit/blob/7473c06517357a61932a28566e788a97ecec3992/core/create/CHANGELOG.md

**question**: should this be dated 2022-05-03 since the file added this part on 3rd of May?

Reply:
when filling in the dates I chose to use the days we actually[ published the changes](https://github.com/Financial-Times/dotcom-tool-kit/releases/tag/create-v2.0.1) rather than the day when release-please generated the changelogs. it's not a big deal that it doesn't recalculate these (there's no way it could really) but if I'm going to add them manually I want to do it properly!